watertap/unit_models/pressure_exchanger.py

```diff
@@ -151,20 +167,18 @@ def build(self):
             has_phase_equilibrium=False)
 
         self.high_pressure_side.add_material_balances(
-            balance_type=self.config.material_balance_type)
+            balance_type=self.config.material_balance_type,
+            has_mass_transfer=self.config.has_mass_transfer)
 
         self.high_pressure_side.add_momentum_balances(
             balance_type=self.config.momentum_balance_type,
             has_pressure_change=True)
 
-        self.high_pressure_side.deltaP.setub(0)
```

Comment:
What was the reasoning to remove this bound? It should still be always true.

Reply:
I think in an earlier pull (that I removed) it was suggested this was redundant.